pkg/prometheus/server/operator.go

```diff
@@ -135,6 +138,35 @@ func New(ctx context.Context, conf operator.Config, logger log.Logger, r prometh
 		kubeletSyncEnabled = true
 	}
 
+	// Check prerequisites for ScrapeConfig
+	verbs := map[string][]string{
+		monitoringv1alpha1.PrometheusAgentName: {"get", "list", "watch"},
```

Comment:
```suggestion
		monitoringv1alpha1.ScrapeConfigtName: {"get", "list", "watch"},
```

Reply:
haha, I was just starting to debug this one, thanks! :D